contracts/LyraL2/LyraPositionHandlerL2.sol

```diff
@@ -116,8 +133,9 @@ contract LyraPositionHandlerL2 is
         return
             (sUSDbalance * NORMALIZATION_FACTOR) /
             USDCPriceInsUSD +
-            IERC20(wantTokenL2).balanceOf(address(this)) +
-            address(this).balance;
+            (IERC20(wantTokenL2).balanceOf(address(this)) *
+                NORMALIZATION_FACTOR) /
+            1e6;
```

Comment:
replace it with `USDC_NORMALIZATION_FACTOR`

Reply:
ser we need the balance in 1e18 there thats why i did that